lib/processor/deps.edn

```diff
@@ -5,7 +5,8 @@
            org.clojure/tools.deps.alpha {:mvn/version "0.12.1120"}
            org.clojure/tools.analyzer   {:mvn/version "1.1.0"}
            ;rewrite-clj/rewrite-clj      {:mvn/version "0.6.1"}
-           hawk/hawk                    {:mvn/version "0.2.11"}
+           com.nextjournal/beholder     {:mvn/version "1.0.0"}
+           org.slf4j/slf4j-nop          {:mvn/version "1.7.36"}
```

Comment:
Why including `org.slf4j/slf4j-nop`? Is it related to `com.nextjournal/beholder`?

Reply:
Yes. From the README (https://github.com/nextjournal/beholder/blob/main/README.md),
```
The core directory-watcher library is designed to have minimal dependencies; currently it only depends on slf4j-api (for internal logging, which can be disabled by passing a NOPLogger in the builder) and jna (for the macOS watcher implementation).
```